discord_api.py

```diff
@@ -45,7 +45,7 @@
 		await channel.send(help(message))
 
 def checkAuthorization(author):
-	if author.name in authorized_authors:
+	if author.name in authorized_authors or author_restrict == False:
 		return True
 	else:
```

Comment:
You can get rid of the `else:` since you're previously returning.

Reply:
Fixed